namespaces/live-1.cloud-platform.service.justice.gov.uk/offender-events-dev/00-namespace.yaml

```diff
@@ -10,3 +10,6 @@ metadata:
     cloud-platform.justice.gov.uk/application: "offender-events"
     cloud-platform.justice.gov.uk/owner: "Digital Prison Services: dps-hmpps@digital.justice.gov.uk"
     cloud-platform.justice.gov.uk/source-code: "https://github.com/ministryofjustice/offender-events.git"
```

Comment:
should be https://github.com/ministryofjustice/prison-offender-events and https://github.com/ministryofjustice/probation-offender-events, with the `dev` version also including https://github.com/ministryofjustice/offender-events-ui